CONTRIBUTING.md

```diff
@@ -217,7 +217,8 @@ We have a set of [coding conventions](CODING-CONVENTIONS.md) to which we try to
 [search-label-enhancement]: https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3APegasysEng%2FPantheon+label%3Aenhancement
 [search-label-bug]: https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3APegasysEng%2FPantheon+label%3Abug
 [search-label-help-wanted]: https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3APegasysEng%2FPantheon+label%3Ahelp%20wanted
-[search-label-good-first-issue]: https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3APegasysEng%2FPantheon+label%3Agood%20first%20issue
+
+[search-label-good-first-issue]:      https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3APegasysEng%2FPantheon+label%3A%22good%20first%20issue%22
```

Comment:
Suggestion:  Consider adding the repo to the search url's path, instead of as query param,  so that you get search UI that is tailored to viewing this repo's issues.  Currently, you get a global github search GUI that can search everything.  Try this for comparison: https://github.com/PegasysEng/pantheon/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22